CMakeLists.txt

```diff
@@ -126,15 +133,18 @@ add_custom_command(
 
 target_include_directories(credentials-fetcherd PUBLIC common)
 
+if(${DISTRO_ID} MATCHES "ubuntu")
+	message(STATUS "Linux distro detected as ubuntu")
+target_link_libraries(credentials-fetcherd
+        PUBLIC systemd boost_program_options krb5 glib-2.0 cf_gmsa_service_private
+        boost_filesystem boost_system crypto protobuf kadm5srv_mit kdb5 gssrpc gssapi_krb5 gssrpc k5crypto com_err krb5support resolv utf8_validity)
+else()
 target_link_libraries(credentials-fetcherd
         PUBLIC systemd boost_program_options krb5 glib-2.0 cf_gmsa_service_private
         boost_filesystem boost_system crypto
         kadm5srv_mit kdb5 gssrpc gssapi_krb5 gssrpc k5crypto
         com_err krb5support resolv)
-
-target_link_libraries(credentials-fetcherd
```

Comment:
Do we need to remove this line for non-ubuntu distros? "api/CMakeLists.txt" file links using same variables for non-ubuntu code path.

Reply:
+1

Reply:
My latest update in the PR now includes the updated Ubuntu 22.04 docker file.